File: 프로그래머스/Level 1/가운데 글자 가져오기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12903?language=python3
def solution(s):
    if len(s) % 2:
        return s[len(s)//2]
    return s[len(s)//2 - 1: len(s)//2 + 1]

File: 프로그래머스/Level 1/가운데 글자 가져오기.cpp
// https://school.programmers.co.kr/learn/courses/30/lessons/12903?language=python3
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    if (s.size() % 2)
    {
        answer += s[s.size()/2];
    } else
    {
        answer += s[s.size()/2 - 1];
        answer += s[s.size()/2];
    }
    return answer;
}